app/code/core/Mage/Core/Helper/Js.php

```diff
@@ -122,28 +121,84 @@ public function getJsSkinUrl($file)
         return Mage::getDesign()->getSkinUrl($file, []);
     }
 
+    /**
+     * Add messages to the JS translation array
+     *
+     * @param string|array $messageText a single or array of messages to translate
+     * @param ?string $module the helper module to use for translating, defaults to 'core'
+     */
+    public function addTranslateData(string|array $messageText, ?string $module)
```

Comment:
```suggestion
    public function addTranslateData(string|array $messageText, ?string $module): string
```

it returns string right?

Reply:
It returns nothing, so `void` would be more appropriate. I was thinking `void` is implied but looks like it's better to be explicit.